platform/mds/mds-web/src/main/java/org/motechproject/mds/web/service/impl/InstanceServiceImpl.java

```diff
@@ -954,6 +962,8 @@ private Object parseValue(ComboboxHolder holder, String methodParameterType, Fie
         } else if (null != holder && holder.isEnumCollection()) {
             String genericType = holder.getEnumName();
             parsedValue = TypeHelper.parse(valueAsString, holder.getTypeClassName(), genericType, classLoader);
+        } else if (parsedValue instanceof UUID) {
```

Comment:
I think we have not to parse UUID value. You should add a condition to the parse method in TypeHelper and just return object, otherwise the unableToParseException will be thrown